Author: Andreas Beckmann <debian@abeckmann.de>
Description: use proper escaping to fix hyphen-used-as-minus-sign lintian warning

--- a/doc/nvidia-settings.1.m4
+++ b/doc/nvidia-settings.1.m4
@@ -25,7 +25,7 @@
 .BI "nvidia\-settings [" "options" "] \-\-glxinfo"
 .PP
 Options:
-.BI "[-vh] [\-\-config=" configfile "] [\-c " ctrl-display "]"
+.BI "[\-vh] [\-\-config=" configfile "] [\-c " ctrl-display "]"
 .br
 .I "         \fB[\-\-verbose=\fP{\fIerrors \fP|\fI warnings \fP|\fI all\fP}\fB]"
 .br
@@ -162,7 +162,7 @@
 file, which controls what X applications should be started when you log into X (or startx), might look something like this:
 .nf
 
-     nvidia-settings --load-config-only &
+     nvidia\-settings \-\-load\-config\-only &
      xterm &
      evilwm
 
@@ -170,8 +170,8 @@
 or:
 .nf
 
-     nvidia-settings --load-config-only &
-     gnome-session
+     nvidia\-settings \-\-load\-config\-only &
+     gnome\-session
 
 .fi
 If you do not already have an
@@ -192,7 +192,7 @@
 with the contents:
 .nf
 
-     nvidia-settings --load-config-only &
+     nvidia\-settings \-\-load\-config\-only &
      . /etc/X11/xinit/xinitrc
 
 .fi
@@ -222,13 +222,13 @@
 Additionally, individual attributes may be specified like this:
 .nf
 
-        nvidia-settings --query CursorShadow
+        nvidia\-settings \-\-query CursorShadow
 
 .fi
 Attributes that may differ per display device (for example, DigitalVibrance can be set independently on each display device when in TwinView) can be appended with a "display device name" within brackets; e.g.:
 .nf
 
-        nvidia-settings --query DigitalVibrance[CRT-0]
+        nvidia\-settings \-\-query DigitalVibrance[CRT\-0]
 
 .fi
 If an attribute is display device specific, but the query does not specify a display device, then the attribute value for all display devices will be queried.
@@ -237,13 +237,13 @@
 to indicate a different X Display; e.g.:
 .nf
 
-        nvidia-settings --query localhost:0.0/DigitalVibrance[DFP-1]
+        nvidia\-settings \-\-query localhost:0.0/DigitalVibrance[DFP\-1]
 
 .fi
 An attribute name may also just be prepended with the screen number and a forward slash:
 .nf
 
-        nvidia-settings --query 0/DigitalVibrance[DFP-1]
+        nvidia\-settings \-\-query 0/DigitalVibrance[DFP\-1]
 
 .fi
 in which case the default X Display will be used, but you can indicate to which X screen to direct the query (if your X server has multiple X screens).
@@ -271,30 +271,30 @@
 For example, the following queries address X screen 0 on the localhost:
 .nf
 
-        nvidia-settings --query 0/VideoRam
-        nvidia-settings --query localhost:0.0/VideoRam
-        nvidia-settings --query [screen:0]/VideoRam
-        nvidia-settings --query localhost:0[screen:0]/VideoRam
+        nvidia\-settings \-\-query 0/VideoRam
+        nvidia\-settings \-\-query localhost:0.0/VideoRam
+        nvidia\-settings \-\-query [screen:0]/VideoRam
+        nvidia\-settings \-\-query localhost:0[screen:0]/VideoRam
 
 .fi
 To address GPU 0 instead, you can use either of:
 .nf
 
-        nvidia-settings --query [gpu:0]/VideoRam
-        nvidia-settings --query localhost:0[gpu:0]/VideoRam
+        nvidia\-settings \-\-query [gpu:0]/VideoRam
+        nvidia\-settings \-\-query localhost:0[gpu:0]/VideoRam
 
 .fi
 See the output of
 .nf
 
-        nvidia-settings --query all
+        nvidia\-settings \-\-query all
 
 .fi
 for what targets types can be used with each attribute.
 See the output of
 .nf
 
-        nvidia-settings --query screens --query gpus --query framelocks --query vcs --query gvis --query fans
+        nvidia\-settings \-\-query screens \-\-query gpus \-\-query framelocks \-\-query vcs \-\-query gvis \-\-query fans
 
 .fi
 for lists of targets for each target type.
@@ -311,9 +311,9 @@
 For example:
 .nf
 
-        nvidia-settings --assign FSAA=2
-        nvidia-settings --assign 0/DigitalVibrance[CRT-1]=9
-        nvidia-settings --assign [gpu:0]/DigitalVibrance=0
+        nvidia\-settings \-\-assign FSAA=2
+        nvidia\-settings \-\-assign 0/DigitalVibrance[CRT\-1]=9
+        nvidia\-settings \-\-assign [gpu:0]/DigitalVibrance=0
 .fi
 .PP
 Multiple queries and assignments may be specified on the command line for a single invocation of
@@ -329,9 +329,9 @@
 .B nvidia\-settings
 will exit after processing the assignments and/or queries.
 In this case, settings contained within the
-.I ~/.nvidia\-settings-rc
+.I ~/.nvidia\-settings\-rc
 configuration file will not be automatically uploaded to the X server, nor will the
-.I ~/.nvidia\-settings-rc
+.I ~/.nvidia\-settings\-rc
 configuration file be automatically updated to reflect attribute assignments made via the
 .B \-\-assign
 option.
@@ -342,21 +342,21 @@
 For example:
 .nf
 
-        nvidia-settings --query FSAA
-        nvidia-settings --query 0/FSAA
-        nvidia-settings --query stravinsky.nvidia.com:0/FSAA
+        nvidia\-settings \-\-query FSAA
+        nvidia\-settings \-\-query 0/FSAA
+        nvidia\-settings \-\-query stravinsky.nvidia.com:0/FSAA
 
 .fi
 In the first two cases, the default X Display will be used, in the second case, the screen from the default X Display can be overridden, and in the third case, the entire default X Display can be overridden.
 .PP
 The same possibilities are available in the
-.I ~/.nvidia\-settings-rc
+.I ~/.nvidia\-settings\-rc
 configuration file.
 .PP
 For example, in a computer lab environment, you might log into any of multiple
 workstations, and your home directory is NFS mounted to each workstation.
 In such a situation, you might want your
-.I ~/.nvidia\-settings-rc
+.I ~/.nvidia\-settings\-rc
 file to be applicable to all the workstations.
 Therefore, you would not want your config file to qualify each attribute with an X Display Name.
 Leave the "Include X Display Names in the Config File" option unchecked on the
@@ -378,12 +378,12 @@
 on the computer stravinsky.nvidia.com, export the display to the computer bartok.nvidia.com, but be configuring the X server on the computer schoenberg.nvidia.com:
 .nf
 
-        nvidia-settings --display=bartok.nvidia.com:0 \\
-            --ctrl-display=schoenberg.nvidia.com:0
+        nvidia\-settings \-\-display=bartok.nvidia.com:0 \\
+            \-\-ctrl\-display=schoenberg.nvidia.com:0
 
 .fi
 If
-.B \-\-ctrl-display
+.B \-\-ctrl\-display
 is not specified, then the X Display to control is what
 .B \-\-display
 indicates.
@@ -444,7 +444,7 @@
 Documentation on the NV-CONTROL extension and additional sample clients are available in the
 .B nvidia\-settings
 source tarball.
-Patches can be submitted to linux-bugs@nvidia.com.
+Patches can be submitted to linux\-bugs@nvidia.com.
 .SS 8. TODO
 There are many things still to be added to
 .B nvidia\-settings,
@@ -464,7 +464,7 @@
 .B nvidia\-settings
 is designed; presumably this would make it easier for people to become familiar with the code base.
 .PP
-If there are other things you would like to see added (or better yet, would like to add yourself), please contact linux-bugs@nvidia.com.
+If there are other things you would like to see added (or better yet, would like to add yourself), please contact linux\-bugs@nvidia.com.
 .SH FILES
 .TP
 .I ~/.nvidia\-settings\-rc
